index.js

```diff
@@ -28,31 +30,49 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drinkWater(num) {
+  console.log("Man I sure am thirsty")
+  console.log("Ahh, that hits the spot")
+  console.log(`${num - 1}`)
+}
```

Comment:
Be careful of the difference between `console.log` and `return` here - the Ruby method returns a value, so this Javascript function should also have a return value.